### ISSUE TITLE
liquid fuels not working in liquid compressor

### ISSUE THREAD

**Issue by Pelkenator** [TeamPneumatic/pnc-repressurized#1195]
### Describe the bug

all of the pneumatic craft fuels aren't working in the ATM9 pack lava works but the rest isn't

### How to reproduce the bug

try inserting every possible way it wont work

### Expected behavior

fuel should be inserted into internal storage tank of liquid compressor

### Which Minecraft version are you using?

1.20

### Which version of PneumaticCraft: Repressurized are you using?

pneumaticcraft-repressurized-6.0.0+mc1.20.1.jar

**Comment by Pelkenator** [comment]
120.1 sorry

**Comment by desht** [comment]
Liquid Compressors work just fine in 1.20.1 with any PNC fuels.  Tested nsertion via liquid hopper, putting a bucket of fuel in the GUI, and simply right-clicking with a fuel bucket (e.g. gasoline).  All work perfectly well.

I'd suggest asking the ATM9 devs if they've made any changes to what fuels the liquid compressor accepts.

**Comment by HipHopHuman** [comment]
@desht I want to chime in here. It might be presumptious of me to assume that you haven't also noticed this trend, but in case you haven't, I've been seeing a few "bug reports" or questions over on the Discord for the last few weeks with issues in PNC that make little sense to me (in the sense that they either contradict my knowledge of PNC or just sound wild), and the one consistent factor in all of these Discord messages is All The Mods from versions 6 to 9. Go over on to the Discord and have a quick search for "ATM<version>" or "All The Mods" and look at the number of questions/issues that turn up.

If you've got some time and a coffee/tea at hand, also have a read of some of them at length. You'll notice it's not just potential changes to recipes that are problems in ATM. You'll see problems for bugs that have already been fixed in PNC in the past, problems for AI not working the way it is supposed to, problems where chunk loading mechanics don't work as advertised and more, and the deeper you look, the weirder the issues become.

I think that ATM might be running one (or maybe several) of those performance mods that tweaks entity AI or messes with vanilla code, thus breaking things in PNC, or they've seriously messed something up somewhere in the configs, or they're running incompatible mods we're not aware of.

**Comment by desht** [comment]
Yeah, I am beginning to suspect ATM is doing something weird. "Performance" mods is my suspicion too.

**Comment by HipHopHuman** [comment]
A user came into the Discord last night with the same issue. This led me to jump into the pack and test it out, and I simply could not reproduce the issue, both on the latest version of ATM9 at the time of this writing and one version prior. All the fuels worked as they should. One concerning thing this user also experienced was that only one bucket of liquid (from another mod) had a fuel tab in JEI - not even Lava, a vanilla item, had one. But my fuel tab worked fine and had all the right information. Other vanilla fuel items like Coal and Charcoal did have fuel tabs in this user's instance though. The little sidebar widget in the compressor itself which lists available fuels listed all the valid fuels in the pack both for me and this user. So I dunno what is going on in ATM9. I did see some mods which I've earmarked as problematic in my own time over the years (mostly anything made by someaddon and some shader mods) but these are mostly all to do with entity AI logic / entity culling and changes to light rendering, so it shouldn't affect fuels
